explain-overview.md

```diff
@@ -57,6 +57,79 @@ The following describes the output of the `EXPLAIN` statement above:
+> +---------------------------------+----------+-----------+-----------------------+------------------------------------------------------------------------------+
+> 
+> -- From v6.4.0:
+>
+> -- You can find the difference between `estRows` of `IndexLookUp_11`、`Selection_10`、`IndexRangeScan_8` and `TableRowIDScan_9` and that before v6.4.0.
+> +---------------------------------+----------+-----------+-----------------------+-----------------------------------------------------------------------------------------------------------------+
+> | id                              | estRows  | task      | access object         | operator info                                                                                                   |
+> +---------------------------------+----------+-----------+-----------------------+-----------------------------------------------------------------------------------------------------------------+
+> | IndexJoin_12                    | 12487.50 | root      |                       | inner join, inner:IndexLookUp_11, outer key:test.t1.a, inner key:test.t2.a, equal cond:eq(test.t1.a, test.t2.a) |
+> | ├─TableReader_24(Build)         | 9990.00  | root      |                       | data:Selection_23                                                                                               |
+> | │ └─Selection_23                | 9990.00  | cop[tikv] |                       | not(isnull(test.t1.a))                                                                                          |
+> | │   └─TableFullScan_22          | 10000.00 | cop[tikv] | table:t1              | keep order:false, stats:pseudo                                                                                  |
+> | └─IndexLookUp_11(Probe)         | 12487.50 | root      |                       |                                                                                                                 |
+> |   ├─Selection_10(Build)         | 12487.50 | cop[tikv] |                       | not(isnull(test.t2.a))                                                                                          |
+> |   │ └─IndexRangeScan_8          | 12500.00 | cop[tikv] | table:t2, index:ia(a) | range: decided by [eq(test.t2.a, test.t1.a)], keep order:false, stats:pseudo                                    |
+> |   └─TableRowIDScan_9(Probe)     | 12487.50 | cop[tikv] | table:t2              | keep order:false, stats:pseudo                                                                                  |
+> +---------------------------------+----------+-----------+-----------------------+-----------------------------------------------------------------------------------------------------------------+
+>
+> -- You can find the difference between `estRows` of `Limit_17`、`IndexReader_21`、`Limit_20` and `IndexRangeScan_19` and that before v6.4.0.
```

Comment:
```suggestion
> -- You can find that the `estRows` column values for `Limit_17`, `IndexReader_21`, `Limit_20`, and `IndexRangeScan_19` since v6.4.0 is different from that before v6.4.0.
```